package.json

```diff
@@ -55,13 +55,14 @@
 		"size": "size-limit",
 		"test": "npm run lint && npm run types && npm run unit && npm run build && npm run size",
 		"types": "tsc --noEmit",
-		"unit": "nyc --reporter=lcovonly --reporter=text --exclude-after-remap=false ava"
+		"unit": "nyc --reporter=lcovonly --reporter=text --exclude-after-remap=false ava",
+		"unit:watch": "nyc --reporter=lcovonly --reporter=text --exclude-after-remap=false ava --watch"
```

Comment:
Not important, but note that you can call one script from another and pass additional parameters.

Reply:
@dani-mp How do you do that? `"unit:watch": "npm run unit --watch"` isn't running it in watch mode.

Reply:
`"unit:watch": "npm run unit -- --watch"`

Reply:
Done ✅